core/tests/performance_framework/perf_domain.py

```diff
@@ -247,32 +251,34 @@ def get_load_event_time_millisecs(self):
     def print_details(self):
         """Helper function to print details for all the events."""
         if self.page_session_stats:
-            print 'Total number of requests: %d' % self.get_request_count()
+            print('Total number of requests: %d' % self.get_request_count())
```

Comment:
Shouldn't this become python_utils.print()? Ditto elsewhere.

Reply:
Done